### DISCUSSION TITLE
Import from copymethat.com

### DISCUSSION THREAD

**Discussion by BrandonG777** [mealie-recipes/mealie#2202] [Feature Request]
Import exported data from copymethat.com. Preferably HTML with images. Would also like to see ratings, favorites, "I made this" and other metadata to migrate.

**Comment by michael-genson** [comment]
See https://github.com/hay-kot/mealie/pull/2212. The only main feature I didn't implement was favorites/starred recipes, just because it's tricky with how we do migrations since they're user-based, not recipe-based. Plus they're _technically_ not the same thing since it's "global" in CMT, whereas with Mealie each user gets their own favorites.

The "I made this" marker doesn't have a timestamp, whereas Mealie's does, so if a recipe is marked as "made" we just say it's made at the time of import.

Pretty sure everything else imports, including notes, rating, image, source URL, etc.

**Reply by tit1** [reply]
this is great to know i have a ton in copy me that.

**Reply by BrandonG777** [reply]
I think I noticed after awhile that descriptions weren't migrated over successfully but probably should have tit1 confirm
